scripts/parseJson.js

```diff
@@ -0,0 +1,168 @@
+const path = require('path')
+const load = require('load-json-file')
+const write = require('write-json-file')
+
+var moduleSidebarList = []
+var completeModules = []
+load.sync(path.join(__dirname, 'tempConfig.json')).forEach(module => {
+  var isHeading = module.kind === 'note'
+  moduleSidebarList.push({
+    isHeading: isHeading,
+    name: module.name,
+    hidden: false
+  })
+  if (!isHeading) {
+    var parent = getParent(module.context.file)
+    completeModules.push({
+      name: module.name,
+      description: concatTags(module.description.children[0].children, true),
```

Comment:
This one is very problematic, might be worth doing a `getDescription` function to handle this.